api/nodes/serializers.py

```diff
@@ -1,42 +1,59 @@
 from rest_framework import serializers as ser
 
-from api.base.serializers import JSONAPISerializer, LinksField, Link
+from api.base.serializers import JSONAPISerializer, LinksField, Link, WaterbutlerLink
 from website.models import Node
 from framework.auth.core import Auth
 
 
 class NodeSerializer(JSONAPISerializer):
 
-    filterable_fields = frozenset(['title', 'description'])
+    category_choices = Node.CATEGORY_MAP.keys()
+    category_choices_string = ', '.join(["'{}'".format(choice) for choice in category_choices])
+    filterable_fields = frozenset(['title', 'description', 'public'])
 
     id = ser.CharField(read_only=True, source='_id')
     title = ser.CharField(required=True)
-    description = ser.CharField(required=False, allow_blank=True)
-    category = ser.ChoiceField(choices=Node.CATEGORY_MAP.keys())
+    description = ser.CharField(required=False, allow_blank=True, allow_null=True)
+    category = ser.ChoiceField(choices=category_choices, help_text="Choices: " + category_choices_string)
```

Comment:
I'm going to stick a TODO in this one if we ever need to do this implementation elsewhere in the API. There's help text of other kinds all over the serializer, and this is pretty clean if it only has to happen once. If not only once, then it'll make sense to subclass the field and generalize it (or submit a patch for DRF if they agree that they made a mistake).